typed-racket-lib/typed-racket/types/subtype.rkt

```diff
@@ -266,7 +266,7 @@
          ;; [(_ (Univ:)) A0]
          ;; error is top and bot
```

Comment:
I think so. I think it means that it is only top now. I'm not sure why it would be bot in the first place, but changing this seems to break more tests, so I'm not sure how this should work.